src/features/categories/category-selection/CategorySelect.module.scss

```diff
@@ -137,3 +177,69 @@ $gap: 2px;
     border-top-right-radius: var(--border-radius-inner);
   }
 }
+
+.sideButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: content-box;
+  color: var(--color-neutral-3);
+}
+
+.plusButton {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  color: #007bff;
+  transform: scale(1.5);
+  padding: var(--spacing-sm);
+
+  &:hover {
+    transform: scale(1.6);
+  }
+
+  svg {
+    width: 2rem;
+    height: 2rem;
+  }
+}
+
+.border {
+  border-bottom: 2px solid #ccc;
+  margin: 10px 0;
+}
+
+.border_input {
+  margin: 10px 0;
+  background-color: var(--color-neutral-5);
+  border-radius: var(--border-radius-inner);
+}
+
+.flexcontainer {
+  display: flex;
+  flex-wrap: wrap; // Allow wrapping of items
+  justify-content: flex-start; // Align items to the left
+  width: 100%; // Full width of the container
+  gap: 5px; // Space between items
+
+  @media (max-width: 768px) {
+    gap: 2px; // Reduce gap for smaller screens
+  }
+}
+
+.flexcontainer > div {
```

Comment:
Selectors that use the `>` operator for generic tags such as `div` are not healthy. The moment the HTML structure changes, the selector can be wrong and cause bugs.
Please use a class instead.